modules/effects/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import * as operators from '@ngrx/operators';
```

Comment:
I'm guessing this is for intellisense?

Reply:
I assume this is so a `@deprecated` JSDoc can be added.